crates/core/src/commands/check.rs

```diff
@@ -309,21 +312,13 @@ pub(crate) fn check_repository<P: ProgressBars, S: Open>(
             let data = match be.read_full(FileType::Pack, &id) {
                 Ok(data) => data,
                 Err(err) => {
-                    // FIXME: This needs different handling, now it prints a full display of RusticError
-                    // Instead we should actually collect and return a list of errors on the happy path
-                    // for `Check`, as this is a non-critical operation and we want to show all errors
-                    // to the user.
-                    error!("Error reading data for pack {id} : {err}");
+                    error!("Error reading data for pack {id} : {}", err.display_log());
                     return;
                 }
             };
             match check_pack(be, pack, data, &p) {
                 Ok(()) => {}
-                // FIXME: This needs different handling, now it prints a full display of RusticError
-                // Instead we should actually collect and return a list of errors on the happy path
-                // for `Check`, as this is a non-critical operation and we want to show all errors
-                // to the user.
-                Err(err) => error!("Pack {id} is not valid: {err}",),
```

Comment:
Are the FIXME not FIXMEs anymore?

I don't know well the `check` cmd so I'm wondering.